BugSplatDotNetStandard/BugSplatPostOptions.cs

```diff
@@ -72,4 +72,10 @@ public class FormDataParam
         /// </summary>
         public string FileName { get; set; } = string.Empty;
     }
+
+    internal class AWSResponse
```

Comment:
Another nit, I think technically this is a BugSplat response that contains an s3 upload url. A better name might be GetPresignedUrlResponse.